drivers/gpio/gpio_emul.c

```diff
@@ -294,17 +294,23 @@ int gpio_emul_input_set_masked_pend(const struct device *port, gpio_port_pins_t
 	}
 
 	if (~config->common.port_pin_mask & mask) {
+		LOG_ERR("%s: Pin not supported port_pin_mask=%x mask=%x", __func__,
```

Comment:
Remove

Reply:
This function is only callable from test code. If it returns an error, don't you want to know why?